mlflow/tensorflow/__init__.py

```diff
@@ -887,6 +968,110 @@ def _log_early_stop_callback_metrics(callback, history, metrics_logger):
         if metric_key is not None:
             metrics_logger.record_metrics(restored_metrics, stopped_epoch + 1)
 
+    def _log_keras_model_with_warning(history, args):
+        try:
+            if log_models:
```

Comment:
If `log_models` is `False`, the rest of this method is a no-op. Can we move the `log_models` check out of this method and check it at the call sites?

I.e.

```
if log_models:
    _log_keras_model(history, args)
```

Reply:
done. thx